Source/Core/S2Cell.js

```diff
@@ -0,0 +1,758 @@
+/* eslint-disable new-cap */
+import Cartesian3 from "./Cartesian3.js";
+import Cartographic from "./Cartographic.js";
+import Check from "./Check.js";
+import defaultValue from "./defaultValue.js";
+import defined from "./defined.js";
+import DeveloperError from "./DeveloperError.js";
+import Ellipsoid from "./Ellipsoid.js";
+import FeatureDetection from "./FeatureDetection.js";
+import RuntimeError from "./RuntimeError.js";
+
+/**
+ * S2
```

Comment:
I just noticed, this comment shows up in the documentation under `S2_MAX_LEVEL` -- could this be moved to the constructor's documentation comment, or turned into a non-JSDoc comment?